install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   mesh: |-
-  {{- if .Values.global.securityEnabled }}
+  {{- if .Values.global.mtlsDefault }}
```

Comment:
could we call it mtlsEnabled?   default is confusing as to on or off...

Reply:
Do you mind doing this in separate PR ? Don't want to go trough another wait :-)

Reply:
I had to rebase, so updated it.